tests/test_submission.py

```diff
@@ -0,0 +1,73 @@
+import os
+from dotenv import load_dotenv
+import pytest
+from unittest.mock import patch, MagicMock
+
+from gradescopeapi.classes.connection import GSConnection
+from gradescopeapi.classes.assignments import Assignment
+
+# Load .env file
+load_dotenv()
+GRADESCOPE_CI_INSTRUCTOR_EMAIL = os.getenv("GRADESCOPE_CI_INSTRUCTOR_EMAIL")
+GRADESCOPE_CI_INSTRUCTOR_PASSWORD = os.getenv("GRADESCOPE_CI_INSTRUCTOR_PASSWORD")
+GRADESCOPE_CI_STUDENT_EMAIL = os.getenv("GRADESCOPE_CI_STUDENT_EMAIL")
+
+def get_account(account_type="instructor"):
+    """Creates a connection and returns the account for testing"""
+    connection = GSConnection()
+    if account_type == "instructor":
+        connection.login(GRADESCOPE_CI_INSTRUCTOR_EMAIL, GRADESCOPE_CI_INSTRUCTOR_PASSWORD)
+    else:
+        raise ValueError("Invalid account type: must be 'instructor'")
+    return connection.account
```

Comment:
Not sure why there is a account_type parameter if it only accepts "instructor" as a valid argument.